spec/unit/spotlight/dor/indexer_spec.rb

```diff
@@ -25,27 +28,211 @@
       'SC0340, Accession 2005-101',
       'SC0340, Accession 2005-101, Box 18'
     ]
-    parsable_exemplars_based_on_actual_data.each do |example|
-      it "parses series number from '#{example}'" do
-        ng_mods = Nokogiri::XML("#{mods_start}#{example}#{mods_end}")
+    exemplars_based_on_feigenbaum.each do |example|
```

Comment:
I wonder if we can clean up some of the boilerplate in these methods, and maybe make the association between the example data and the expected result a little more obvious. Here's something I started playing around with: 

```ruby

  let(:mods_loc_phys_loc) do
    Nokogiri::XML <<-EOF
      <mods xmlns="#{Mods::MODS_NS}">
        <location>
          <physicalLocation>#{example}</physicalLocation>
        </location>
      </mods>
    EOF
  end

  let(:mods_rel_item_loc_phys_loc) do
    Nokogiri::XML <<-EOF
      <mods xmlns="#{Mods::MODS_NS}">
        <relatedItem>
          <location>
            <physicalLocation>#{example}</physicalLocation>
          </location>
        </relatedItem>
      </mods>
    EOF
  end

  before do
    allow(r).to receive(:harvestdor_client)
  end

  describe '#add_series' do
    {
      'Call Number: SC0340, Accession 2005-101' => '2005-101',
      'Call Number: SC0340, Accession 2005-101, Box : 39, Folder: 9' => '2005-101',
      'Call Number: SC0340, Accession: 2005-101' => '2005-101',
      'Call Number: SC0340, Accession: 2005-101, Box : 50, Folder: 31' => '2005-101',
      'SC0340, Accession 2005-101' => '2005-101',
      'SC0340, Accession 2005-101, Box 18' => '2005-101',
      'MSS Photo 451, Series 1, Box 32, Folder 11, Sleeve 32-11-2, Frame B32-F11-S2-6' => '1',
      'Series 1, Box 10, Folder 8' => '1',
      'Collection: M1090 , Series: 1 , Box: 5 , Folder: 42' => '1',
      'Series Biographical Photographs | Box 1 | Folder Abbot, Nathan' => 'Biographical Photographs',
      'Series General Photographs | Box 1 | Folder Administration building--Outer Quad' => ' General Photographs',
      'SC0340, 1986-052, Box 18' => nil,
      'SC0340' => nil,
      'Stanford University. Libraries. Department of Special Collections and University Archives' => nil
    }.each do |example, expected|
      describe "for example '#{example}'" do
        let(:example) { example }

        before do
          subject.send(:add_series, sdb, solr_doc)
        end

        context 'in the physicalLocation' do
          before do
            allow(r).to receive(:mods).and_return(mods_loc_phys_loc)
          end

          it 'has the expected series' do
            expect(solr_doc['series_ssi']).to eq expected
          end
        end

        context 'in a related item physicalLocation' do
          before do
            allow(r).to receive(:mods).and_return(mods_rel_item_loc_phys_loc)
          end

          it 'has the expected series' do
            expect(solr_doc['series_ssi']).to eq expected
          end
        end
      end
    end
```

There's probably room to improve this, even, and might be something we can package up for other steps to take advantage of too.